test/liblll/Parser.cpp

```diff
@@ -180,6 +181,290 @@ BOOST_AUTO_TEST_CASE(macro_with_zero_args)
+    "{ (dup12 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (dup13 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (dup14 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (dup15 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (dup16 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (SWAP1 0xff 0xff) (return 0x0 0x0)}",
+    "{ (swap2 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (swap3 0xff 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (swap4 0xff 0xff 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (swap5 0xff 0xff 0xff 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (swap6 0xff 0xff 0xff 0xff 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (swap7 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff) (return 0x0 0x0)}",
+    "{ (swap8 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0x00) (return 0x0 0x0)}",
+    "{ (swap9 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0x00) (return 0x0 0x0)}",
+    "{ (swap10 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0x00) (return 0x0 0x0)}",
+    "{ (swap11 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0x00) (return 0x0 0x0)}",
+    "{ (swap12 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0x00) (return 0x0 0x0)}",
+    "{ (swap13 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0x00) (return 0x0 0x0)}",
+    "{ (swap14 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0x00) (return 0x0 0x0)}",
+    "{ (swap15 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0x00) (return 0x0 0x0)}",
+    "{ (swap16 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0xff 0x00) (return 0x0 0x0)}",
+    "{ (LOG0 0 0) (return 0x0 0x0) }",
+    "{ (LOG1 0 0 0xff) (return 0x0 0x0) }",
+    "{ (log2 0 0 0xff 0xff) (return 0x0 0x0) }",
+    "{ (log3 0 0 0xff 0xff 0xff) (return 0x0 0x0) }",
+    "{ (log4 0 0 0xff 0xff 0xff 0xff) (return 0x0 0x0) }",
+    "{ (CREATE 0xff 0x0 0x0) (return 0x0 0x0) }",
+    "{ (call 100 0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6 23 0 0 0 0) (return 0x0 0x0) }",
+    "{ (CALLCODE 100 0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6 0 0 0 0 0) (return 0x0 0x0) }",
+    "{ (RETURN 0x0 0x0) }",
+    "{ (DELEGATECALL 100000 0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6 0 0 0 0) (return 0x0 0x0) }",
+    "{ (STATICCALL 10000 0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6 0 0 0 0) (return 0x0 0x0) }",
+    "{ (REVERT 0x0 0x0) (return 0x0 0x0)}",
+    "{ (SELFDESTRUCT (ORIGIN)) }"
+  };
+
+  for (size_t i = 0; i < opcodes_lll.size(); i++) {
+
+    std::vector<std::string> errors;
+    bytes code = eth::compileLLL(opcodes_lll[i], false, &errors);
+
+    if (!errors.empty()) {
+      BOOST_FAIL(opcodes_lll[i]);
+    }
```

Comment:
For this part above you could copy the code from `test/liblll/ExecutionFramework.h`.